src/salesforce_uploader.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import { fetchConfig } from './lib/config'
+import AWS from 'aws-sdk'
+import NamedError from './lib/NamedError'
+import { authenticate } from './lib/salesforceAuthenticator'
+
+const s3 = new AWS.S3({ signatureVersion: 'v4' })
+const BUCKET = 'fulfilment-output-test'
+
+export function handler (input:?any, context:?any, callback:Function) {
+  if (input == null || typeof input.path !== 'string') {
+    callback(new NamedError('inputError', 'Input did not contain filename'))
+    return
+  }
+  uploader(input).then((r) => {
+    console.log(r)
+    console.log('success')
+    callback(null, r)
+  }).catch(e => {
+    console.log('oh no  ')
+    console.log(e)
+    callback(e)
+  })
+}
+
+async function uploader (input: { path: string }) {
+  const config = await fetchConfig()
+  const salesforce = await authenticate(config)
+  console.log('Finding fulfilment folder.')
+  const folder = await salesforce.getFolderId('HOME_DELIVERY_FULFILMENT')
```

Comment:
as discussed we should save the folder id in the configuration file instead of doing this additional api call